app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,11 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+.container {
+  margin-top: 10px;
+}
+
+.card, .field, .btn {
+  margin-bottom: 10px;
+}
+
+.form-class {
+  width: 400px;
+}
```

Comment:
I think we should apply some convention here. Because currently all styles are available in every view. What if something has the same class, but we want different styles? I also suggest using a Sass preprocessor, it'll look better. 👍